### PR TITLE
[DR-1096] Persist preferred language on local storage

### PR DESCRIPTION
**Pull request by alegarciamina**

### CODE REVIEW

File: src/wwwroot/controllers/MainCtrl.js [hunk 1]
```diff
@@ -14,11 +14,16 @@
...
+    '$translate',
+    'utils'
   ];
 
-  function MainCtrl($rootScope, $scope, $window, $location, auth, $log, ModalService, $route, $interval) {
-    
+  function MainCtrl($rootScope, $scope, $window, $location, auth, $log, ModalService, $route, $interval, $translate, utils) {
+     
+    var key = utils.getPreferredLanguage();
+    $translate.use(key);
```
Comment (andresmoschini) [inline src/wwwroot/controllers/MainCtrl.js]: Is this code executed only one time on page loading or on each navigation?

Comment (andresmoschini) [inline src/wwwroot/controllers/MainCtrl.js]: What happen when _key_ is null? should we deal with that scenario?

Comment (alegarciamina) [reply src/wwwroot/controllers/MainCtrl.js]: It's executed on page loading only, just once.

Comment (alegarciamina) [reply src/wwwroot/controllers/MainCtrl.js]: If key is null is just ignored and use English as default language.

**Pull request merged**